dewey/core/urls/api_v1.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework_swagger.views import get_swagger_view
+from django.conf.urls import include, url
+
+from dewey.environments.api import urls_v1 as environments_api
+from dewey.networks.api import urls_v1 as networks_api
+from dewey.salt.api import urls_v1 as salt_api
+
+app_name = 'api_v1'
+schema_view = get_swagger_view(title='Dewey API')
+
+urlpatterns = [
```

Comment:
Nit: Would have expected to have the base part of the path here for each model, but I'm not sure if that is best practice or not (my Django is rusty at best)

Reply:
Yeah, that prefix is actually included in the routes that DRF generates.  Adding it again here would result in redundant paths like `/api/networks/networks/assignments`.